plugins/jira-dashboard-backend/README.md

```diff
@@ -73,6 +73,28 @@ metadata:
     jira.com/project-key: separate-jira-instance/my-project-key
 ```
 
+### Custom Jira Filters
+
+You can now define custom Jira filters directly in your `app-config.yaml` file. This allows you to create and display filters beyond the ones provided by the plugin.
```

Comment:
We can remove the "now" here. Otherwise really nice docs! :+1:

Reply:
Done👍